README.md

```diff
@@ -0,0 +1,54 @@
+# Toronto JS Style Guide
+This is the project for Toronto JS' Style Guide. It contains the documentation for how to use and create graphical assets for the community, following a set of guidelines.
+
+## Rquirements
+- Node.js v18.0.0 or higher
+- Local certificate
+
+## Local certificate
+You will need to create a local certificate for testing locally. The files needed are: `certs/server.crt` and `certs/server.key`.
+
+To get those files you can use [`mkcert`](https://github.com/FiloSottile/mkcert) as it is the easiest way to create a local certificate.
+
+You can run the following command:
+```shell
+$ mkcert -key-file certs/server.key -cert-file certs/server.crt localhost
```

Comment:
@usrrname for generating the certificates run this command on the project home

Reply:
yep that's what I did, I think it might just be a M2 thing